main/src/ca/uwaterloo/flix/language/ast/Symbol.scala

```diff
@@ -133,6 +133,13 @@ object Symbol {
     new ClassSym(ns.parts, ident.name, ident.loc)
   }
 
+  /**
+    * Returns the instance symbol for the given name `ident` in the given namespace `ns`.
+    */
+  def mkInstanceSym(ns: NName, ident: Ident): InstanceSym = {
```

Comment:
I am not sure I understand why instances would have names? Are these purely internal names? Where would you be able to refer to them?